app/Http/Controllers/UserController.php

```diff
@@ -23,16 +23,18 @@ public function store(Request $request)
         $request->validate([
             'first_name' => 'required',
             'last_name' => 'required',
-            'email' => 'required',
+            'email' => 'required|unique:users',
             'password' => 'required',
             'description' => '',
```

Comment:
```suggestion
            'description' => 'nullable',
```

According to the [Laravel Documentation](https://laravel.com/docs/8.x/validation#a-note-on-optional-fields) a optional field should look like this